cloud/fallback-public-cloud.yaml

```diff
@@ -186,6 +198,10 @@ clouds:
         storage-endpoint: https://core.windows.net
         identity-endpoint: https://graph.windows.net        
       koreasouth:
+        endpoint: https://management.azure.com
+        storage-endpoint: https://core.windows.net
+        identity-endpoint: https://graph.windows.net
+      francecentral:
```

Comment:
seems like we're missing francesouth?
are any of the newly announced african ones etc alive yet?

Reply:
Have just tried francesouth again. Even though it's listed as generally available, the Azure API isn't very happy

```
$ juju bootstrap azure/francesouth
Creating Juju controller "azure-francesouth" on azure/francesouth
ERROR getting instance types: listing VM sizes: compute.VirtualMachineSizesClient#List: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="NoRegisteredProviderFound" Message="No registered resource provider found for location 'francesouth' and API version '2016-04-30-preview' for type 'locations/vmSizes'. The supported api-versions are '2015-05-01-preview, 2015-06-15, 2016-03-30, 2016-04-30-preview, 2016-08-30, 2017-03-30, 2017-12-01, 2018-04-01, 2018-06-01, 2018-10-01'. The supported locations are 'eastus, eastus2, westus, centralus, northcentralus, southcentralus, northeurope, westeurope, eastasia, southeastasia, japaneast, japanwest, australiaeast, australiasoutheast, brazilsouth, southindia, centralindia, westindia, canadacentral, canadaeast, westus2, westcentralus, uksouth, ukwest, koreacentral, koreasouth, francecentral'."
```

Reply:
have kept the francesouth region in. from the [status page](https://azure.microsoft.com/en-us/status/), it looks as though it--along with the German and other [newly announced regions](https://azure.microsoft.com/en-us/global-infrastructure/regions/)--offers fewer services than most of the other regions, which could explain why the API is saying it doesn't exist.